spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1875,8 +1875,8 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
               + s"(SELECT c1, cast(make_interval(c0, c1, c0, c1, c0, c0, c2) as string) as casted from $table) as B "
               + "where A.c1 = B.c1 ",
             Set(
-              "Comet shuffle is not enabled: spark.comet.exec.shuffle.enabled is not enabled",
-              "make_interval is not supported")),
+              "Cast from CalendarIntervalType to String is not supported",
+              "Comet shuffle is not enabled: spark.comet.exec.shuffle.enabled is not enabled")),
```

Comment:
```suggestion
              "Comet shuffle is not enabled. To enable set spark.comet.exec.shuffle.enabled")),
```

Reply:
This message comes from `CometExecRule`. The message format wasn't modified as part of this PR, so changing the expected message here seems out of scope for this PR?